stubs/boltons/boltons/cacheutils.pyi

```diff
@@ -0,0 +1,107 @@
+from _typeshed import Incomplete
+from collections.abc import Callable, Generator
+from typing import Any, NoReturn, TypeVar
+
+_KT = TypeVar("_KT")
+_VT = TypeVar("_VT")
+
+xrange = range
+basestring: Any
+PREV: int
+NEXT: int
+KEY: int
+VALUE: int
+DEFAULT_MAX_SIZE: int
+
+class LRI(dict[_KT, _VT]):
+    hit_count: int
+    max_size: int
+    on_miss: Callable[[_KT], _VT]
+    def __init__(self, max_size: int = ..., values: Incomplete | None = ..., on_miss: Incomplete | None = ...) -> NoReturn: ...
+    def __setitem__(self, key: _KT, value: _VT) -> NoReturn: ...
+    def __getitem__(self, key: _KT) -> _VT: ...
+    def get(self, key, default: Incomplete | None = ...): ...
+    def __delitem__(self, key: _KT) -> NoReturn: ...
+    def pop(self, key, default=...): ...
+    def popitem(self): ...
+    def clear(self) -> None: ...
+    def copy(self): ...
+    def setdefault(self, key, default: Incomplete | None = ...): ...
+    def update(self, E, **F) -> None: ...  # type: ignore
```

Comment:
E and F seem undefined, why is the error ignored?

Reply:
I just don't know how to type it

Reply:
I have no idea how to annotate it correctly.

Reply:
Then we should just annotate it with `*args`/`**kwargs` for now. I want to avoid type ignores.